extensions-core/s3-extensions/pom.xml

```diff
@@ -115,8 +115,8 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk-sts</artifactId>
-      <scope>provided</scope>
```

Comment:
why this change?

Reply:
`aws-common` doesn't contain it, surprisingly Kinesis extension contains it, when Kinesis ext. is removed then the S3 ext. fails.